### PR TITLE
Use trunk-analytics-uploader action

### PR DESCRIPTION
**Pull request by riya-n**
- refer to pre-release tag used here: https://github.com/trunk-io/trunk-analytics-uploader/releases/tag/v0.1.0
- also we would probably need to make https://github.com/trunk-io/trunk-analytics-uploader public

### CODE REVIEW

File: .github/workflows/trunk-hourly-e2e-tests.yml [hunk 1]
```diff
@@ -126,9 +125,13 @@ jobs:
 
       - run: ls -l ./cypress/reports/mochareports/
       - name: Upload results
-        run: ./.trunk-ci-analytics/bin --token=${{ secrets.TRUNK_DEBUGGER_TOKEN }}  --input="cypress/reports/**/*junit*.xml"
+        uses: trunk-io/trunk-analytics-uploader@v0.1.0
+        with:
+          junit_paths: cypress/reports/**/*junit*.xml
+          org_url_slug: trunk-staging-org
+          token: ${{ secrets.TRUNK_API_TOKEN }}
```
Comment (riya-n) [inline .github/workflows/trunk-hourly-e2e-tests.yml:132]: Why were we using the `TRUNK_DEBUGGER_TOKEN ` earlier? Is the `TRUNK_API_TOKEN` not set up?

File: .github/workflows/trunk-hourly-e2e-tests.yml [hunk 2]
```diff
@@ -126,9 +125,13 @@ jobs:
 
       - run: ls -l ./cypress/reports/mochareports/
       - name: Upload results
-        run: ./.trunk-ci-analytics/bin --token=${{ secrets.TRUNK_DEBUGGER_TOKEN }}  --input="cypress/reports/**/*junit*.xml"
+        uses: trunk-io/trunk-analytics-uploader@v0.1.0
+        with:
+          junit_paths: cypress/reports/**/*junit*.xml
+          org_url_slug: trunk-staging-org
```
Comment (riya-n) [inline .github/workflows/trunk-hourly-e2e-tests.yml:131]: We still want to use our org slug right?

File: .github/workflows/trunk-hourly-e2e-tests.yml (hunk 1 above)
Comment (zaycev) [reply .github/workflows/trunk-hourly-e2e-tests.yml:132]: This is what old binary expected to use. We can't assume how token env var will be named in our users setup, that way we can always pass `TRUNK_DEBUGGER_TOKEN` or any other value with `--token` key.

File: .github/workflows/trunk-hourly-e2e-tests.yml (hunk 2 above)
Comment (zaycev) [reply .github/workflows/trunk-hourly-e2e-tests.yml:131]: Yep.

File: .github/workflows/trunk-hourly-e2e-tests.yml [hunk 3]
```diff
@@ -126,9 +125,12 @@ jobs:
...
+          token: ${{ secrets.TRUNK_DEBUGGER_TOKEN }}
         env:
-          REPO_URL: ${{ github.repositoryUrl }}
```
Comment (riya-n) [inline .github/workflows/trunk-hourly-e2e-tests.yml:131]: Will the CLI auto-detect this or do we need to keep the REPO_URL env var here

Comment (zaycev) [reply .github/workflows/trunk-hourly-e2e-tests.yml:131]: Yeah, env is not needed. Cli should auto-discover this URL.

**Pull request merged**